src/hooks/useSavingsAccount.js

```diff
@@ -114,7 +114,7 @@ export default function useSavingsAccount(includeDefaultDai) {
       client: compoundClient,
       pollInterval: COMPOUND_QUERY_INTERVAL,
       skip: !hasAccountAddress,
-      variables: { id: toLower(accountAddress) },
+      variables: { id: accountAddress?.toLowerCase() },
```

Comment:
as lodash methods are null safe, and we have not covered all files by typescript so sometimes it could be "just in case".  
in this case I added it because the line above we have `const hasAccountAddress = !! accountAddress`